tests/product/prestoadmin_installer.py

```diff
@@ -69,84 +57,16 @@ def assert_installed(testcase, msg=None):
-                os.path.join(
-                    installer_container.get_local_mount_dir(container_name),
-                    'presto-admin'),
-                ignore=shutil.ignore_patterns('tmp', '.git', 'presto*.rpm')
-            )
-
-            # Pin pip to 7.1.2 because 8.0.0 removed support for distutils
-            # installed projects, of which the system setuptools is one on our
-            # Docker image. pip 8.0.1 or 8.0.2 replaced the error with a
-            # deprecation warning, and also warns that Python 2.6 is
-            # deprecated. While we still need to support Python 2.6, we'll pin
-            # pip to a 7.x version, but we should revisit this once we no
-            # longer need to support 2.6:
-            # https://github.com/pypa/pip/issues/3384
-            installer_container.run_script_on_host(
-                'set -e\n'
-                'pip install --upgrade pip==7.1.2\n'
-                'pip install --upgrade wheel==0.23.0\n'
-                'pip install --upgrade setuptools==20.1.1\n'
-                'mv %s/presto-admin ~/\n'
-                'cd ~/presto-admin\n'
-                'make %s\n'
-                'cp dist/prestoadmin-*.tar.bz2 %s'
-                % (installer_container.mount_dir,
-                   'dist' if online_installer else 'dist-offline',
-                   installer_container.mount_dir),
-                container_name)
-
-            try:
-                os.makedirs(cluster.get_dist_dir(unique))
-            except OSError, e:
-                if e.errno != errno.EEXIST:
-                    raise
-            local_container_dist_dir = os.path.join(
-                prestoadmin.main_dir,
-                installer_container.get_local_mount_dir(container_name)
-            )
-            installer_file = fnmatch.filter(
-                os.listdir(local_container_dist_dir),
-                'prestoadmin-*.tar.bz2')[0]
-            shutil.copy(
-                os.path.join(local_container_dist_dir, installer_file),
-                cluster.get_dist_dir(unique))
-        finally:
-            installer_container.tear_down()
-
-    @staticmethod
-    def _copy_dist_to_host(cluster, local_dist_dir, dest_host):
-        for dist_file in os.listdir(local_dist_dir):
+    def copy_dist_to_master(self, cluster, dist_dir):
+        if (not os.path.isdir(dist_dir) or
+                len(fnmatch.filter(
+                    os.listdir(dist_dir),
+                    'prestoadmin-*.tar.bz2')) == 0):
+            self.testcase.fail(
+                'Unable to find presto-admin package. Have you run one of '
+                '`make dist*` or `make docker-dist*` command?')
+        for dist_file in os.listdir(dist_dir):
             if fnmatch.fnmatch(dist_file, "prestoadmin-*.tar.bz2"):
                 cluster.copy_to_host(
-                    os.path.join(local_dist_dir, dist_file),
-                    dest_host)
+                    os.path.join(dist_dir, dist_file),
```

Comment:
above this `if` block there is a check if there is a file in this directory and fails if there isn't